Python/tier2_redundancy_eliminator_cases.c.h

```diff
@@ -1674,6 +1672,18 @@
             break;
         }
 
+        case _COLD_EXIT: {
```

Comment:
Add this file to the generated files in `.gitattributes`?

Reply:
But under the wrong name, I'll fix that.